server/server.js

```diff
@@ -153,7 +153,13 @@ app.post(`${apiBaseUrl}/approve-user`,
   keycloak.allowRolesMiddleware('ministry_of_health'),
   asyncMiddleware(async (req, res) => {
     await validate(AccessRequestApproval, req.body);
-    await keycloak.approvePendingRequest(req.body.userId, req.body.role, [req.body.region]);
+    const results = await dbClient.db[collections.EMPLOYER_SITES].findDoc({
+      or: req.body.sites.map((siteId) => ({ 'siteId ilike': `%${siteId}%` })),
```

Comment:
We're using `ilike` quite a bit throughout the app. Shouldn't this be a strict match on something like IDs.